spec/factories/authorization_requests.rb

```diff
@@ -199,7 +199,7 @@
     trait :with_scopes do
       after(:build) do |authorization_request, evaluator|
         if authorization_request.need_complete_validation? || evaluator.fill_all_attributes
-          next if authorization_request.scopes.any?
+          next if authorization_request.scopes.present?
```

Comment:
imo tu peux revert ici (`present?` n'est pas assez permissif), surtout que ça ne touche plus du tout ta PR

```suggestion
          next if authorization_request.scopes.any?
```

Reply:
Ah non c'est bon, j'avais pas lu le bon bout de code !